File: AngularLovelyjubbly.Data.Sql.Models/RegularSeasonWins.cs
namespace AngularLovelyjubbly.Data.Sql.Models
{
    public partial class RegularSeasonWins
    {
        public RegularSeasonWins()
        {
        }

        public int RegularSeasonWinsId { get; set; }
        //foreign key
        public int TeamId { get; set; }
        //navigation property
        public virtual Team Team { get; set; }
        //foreign key
        public int TournamentId { get; set; }
        //navigation property
        public virtual Tournament Tournament { get; set; }
        public decimal Wins { get; set; }
    }
}

File: AngularLovelyjubbly/ClientApp/src/app/shared/models/regularseasonwins.ts
import { ITournament } from './tournament';
import { ITeam } from './team';

export interface IRegularSeasonWins {
    regularSeasonWinsId: number;
    tournament: ITournament;
    team: ITeam;
    wins: number;
}

export class RegularSeasonWins implements IRegularSeasonWins {

    constructor(public regularSeasonWinsId: number, public tournament: ITournament,
        public team: ITeam, public wins: number) {
    }
}
